### DISCUSSION TITLE
How to use features requiring RTTI and exceptions?

### DISCUSSION THREAD

**Discussion by mzient** [NVIDIA/cccl#1029] [libcu++]
The file `include/cuda/std/detail/__config` unconditionally disables multiple features, including RTTI and exceptions. If this header is pulled in, the subsequently included headers will have these features disabled.

**Comment by wmaxey** [comment]
Possibly addressed with this branch.

https://github.com/NVIDIA/libcudacxx/tree/feature/rtti_and_exception_exposure

**Comment by wmaxey** [comment]
Moving to backlog as #158 will include this change.

@mzient Please let me know if you have any issues with exception/RTTI support on any specific compilers.

**Comment by wmaxey** [comment]
Moving to a discussion as this is not precisely a feature request.
